std/math.d

```diff
@@ -1580,8 +1587,18 @@ real exp(real x) @trusted pure nothrow @nogc
         enum real C2 = 1.428606820309417232121458176568075500134E-6L;
 
         // Overflow and Underflow limits.
-        enum real OF =  1.1356523406294143949492E4L;
-        enum real UF = -1.1432769596155737933527E4L;
+        static if (real.mant_dig == 64) // 80-bit reals
+        {
+            enum real OF =  1.1356523406294143949492E4L; // ln((1-2^-64) * 2^16384) ≈ 0x1.62e42fefa39efp+13
+            enum real UF = -1.1398805384308300613366E4L; // ln(2^-16445) ≈ -0x1.6436716d5406ep+13
```

Comment:
Cephes.  It's documented at the top of the module.  FYI, the 64-bit real equivalents would be:
```
enum real OF = 7.09782712893383996732E2L;
enum real UF = -7.451332191019412076235E2L;
```